Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -288,6 +289,7 @@ local questsToBlacklistBySoDPhase = {
         [90146] = true, -- Hiding Druid Mangle Mulgore for now as there are too many icons
         [90147] = true, -- Hiding Paladin Hand of Reckoning Westfall for now as there are too many icons
         [90148] = true, -- Hiding Paladin Exorcist Duskwood for now as there are too many icons
+        [90151] = true, -- Hiding Sniper Training Westfall for now as there are too many icons
```

Comment:
The spawns need to be cleaned up. The scout only spawns in two locations.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5453/commits/45922c2b1b46f2cda9fe9f61f059a42992a1bc99